shell/protoc.sh

```diff
@@ -152,6 +152,19 @@ if has_grpc_client "node"; then
   )
 
   "$grpc_tools_node_bin" "${node_args[@]}" "$(get_repo_directory)/api/"*.proto
+
+  npm install -g grpc_tools_node_protoc_ts
```

Comment:
IDEA

we decided to go with "Just Install" option.
npm is good enough to handle it and `npm list` might be quite slow

so ... we can drop `if ! npm list -g --prefix "$NODE_GRPC_TOOLS_CACHE_DIR" | grep grpc-tools@"$node_tools_version"`

e.g. with just 3 packages:
```
time npm list -g
/Users/karlismelderis/.asdf/installs/nodejs/16.13.0/.npm/lib
├── grpc_tools_node_protoc_ts@5.3.2
├── npm@8.5.0
└── yarn@1.22.17

npm list -g  0.61s user 0.29s system 105% cpu 0.855 total
```
and install is always around 1-2 seconds:
```
time npm install -g grpc_tools_node_protoc_ts

changed 8 packages, and audited 9 packages in 661ms

found 0 vulnerabilities
Reshimming asdf nodejs...
npm install -g grpc_tools_node_protoc_ts  1.23s user 1.37s system 90% cpu 2.885 total
```